connector/saml/saml.go

```diff
@@ -252,16 +258,21 @@ func (p *provider) POSTData(s connector.Scopes, id string) (action, value string
 		},
 		AssertionConsumerServiceURL: p.redirectURI,
 	}
+
+	logStr := "saml: no authn request entity issuer set"
```

Comment:
Don't care for the `logStr` pattern. Probably fine to just have two log lines and an if-else.